File: Chapter 7/Ruby/page_objects/sidebar_cart.rb
class SidebarCart
  def initialize(selenium)
    @selenium = selenium
  end 
  
  def summary
    @selenium.find_element(:id, "Cart66WidgetCartEmptyAdvanced").text
  end
  
  def subtotal
     @selenium.find_element(:class, "Cart66Subtotal").text
  end     
end                                           

File: Java/src/test/java/chapter7/pageObjects/shared/sidebar/SidebarCart.java
package chapter7.pageObjects.shared.sidebar;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class SidebarCart {

  private WebDriver selenium;

  public SidebarCart(WebDriver selenium){
    this.selenium = selenium;
  }

  public String getSummary(){
    return selenium.findElement(By.id("Cart66WidgetCartEmptyAdvanced")).getText();

  }

  public String getSubtotal(){
    return selenium.findElement(By.className("Cart66Subtotal")).getText();
  }


}
